package.json

```diff
@@ -15,6 +15,7 @@
     "@react-hookz/web": "^23.1.0",
     "@tanstack/react-query": "^4.32.0",
     "bignumber.js": "^9.1.1",
+    "ethers": "^6.7.1",
```

Comment:
adding ethers when we have viem already is definitely adding a lot of redundant code.

Check [this out](https://wagmi.sh/core/actions/switchNetwork)

Reply:
I don't think you are even using it 😅